open_schools_platform/parent_management/families/models.py

```diff
@@ -19,7 +18,7 @@ def create_family(self, name: str):
 class Family(BaseModel):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4)
     parent_profiles = models.ManyToManyField(ParentProfile, related_name="parent_families")
-    student_profiles = models.ManyToManyField(StudentProfile, related_name="student_families")
+    student_profiles = models.ManyToManyField('student.StudentProfile', related_name="student_families")
```

Comment:
I needed to change it this way because I wanted to get rid of importing StudentProfile model there. I had to do so because I had circular import issue.